config/jobs/kubernetes/sig-network/ingress-gce-e2e.yaml

```diff
@@ -180,7 +180,6 @@ periodics:
       - --env=GCE_ALPHA_FEATURES=NetworkEndpointGroup
       - --env=KUBE_GCE_ENABLE_IP_ALIASES=true
       - --extract=ci/latest
-      - --gcp-project-type=ingress-project
```

Comment:
Did we check to make sure these don't have larger quotas?

Reply:
the one with large quota was `ci-ingress-gce-e2e-scale` IIRC

 We are in a try and see mode, but we can always go back